lib/generators/manageiq/provider/templates/app/models/%plugin_path%/inventory/persister.rb

```diff
@@ -1,19 +1,3 @@
 class <%= class_name %>::Inventory::Persister < ManageIQ::Providers::Inventory::Persister
   require_nested :<%= manager_type %>
-
-  def strategy
-    nil
-  end
```

Comment:
This comes from the base Persister class [here](https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/inventory/persister.rb#L55-L57)